setup.py

```diff
@@ -41,6 +41,8 @@
         'caiman': ['h5py==3.3.0',
                    'scipy==1.7.0',
                    'caiman@git+https://github.com/flatironinstitute/CaImAn.git@v1.8.9'],
+        'fissa': ['fissa@https://github.com/rochefort-lab/fissa.git'],
```

Comment:
```suggestion
        'fissa': ['fissa@git+https://github.com/rochefort-lab/fissa.git'],
```

Reply:
Hi @tdincer, is the `git+` prefix required?  In my previous testing, I believe it is required but I could be mistaken.

Reply:
You are right `git+` prefix is required.